app/src/main/java/de/westnordost/streetcomplete/quests/collection_times/AddCollectionTimes.java

```diff
@@ -0,0 +1,56 @@
+package de.westnordost.streetcomplete.quests.collection_times;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.text.TextUtils;
+import android.util.Log;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.osm.Countries;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+
+public class AddCollectionTimes extends SimpleOverpassQuestType
+{
+	@Inject public AddCollectionTimes(OverpassMapDataDao overpassServer) { super(overpassServer); }
+
+	@Override protected String getTagFilters()
+	{
+		String[] amenities = { "post_box" };
+
+		return " nodes, ways, relations with ( amenity ~ "
+				+ TextUtils.join("|", amenities) + ")" +
+				" and !collection_times" + " and (access !~ private|no)"; // exclude ones without access to general public
```

Comment:
Can be simplified to `nodes with amenity=post_box and !collection_times and (access !~ private|no)`
I would not expect a post box to be a way or relation...

Reply:
I would also not expect a post box to be private.

Reply:
What about post box in for example military base? Or gated community?